scripts/credential-manager/kubernetes-credential-manager.sh

```diff
@@ -1,13 +1,40 @@
 #!/bin/sh -xe
 
-WAIT_APPROVED_SEC=${WAIT_APPROVED_SEC:-600}
-CSR_NAME=${CSR_NAME:-nuvlaedge-csr}
+WAIT_APPROVED_SEC=${WAIT_APPROVED_SEC:-600}                                                                        
 
 SHARED="/srv/nuvlaedge/shared"
 SYNC_FILE=".tls"
 CA="/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
 USER="nuvla"
 
+echo "The NuvlaEdge UUID is set to: ${NUVLAEDGE_UUID}"
+echo "The MY_NAMESPACE is set to: ${MY_NAMESPACE}"
```

Comment:
This seems to be the only place where `MY_NAMESPACE` is used.
But later on there is a `NAMESPACE` generated from `NUVLAEDGE_UUID` or random characters.

Reply:
I have removed the not-useful echo statement.
The later NAMESPACE is set to be generated from six random characters.